include/swift/Sema/ConstraintLocator.h

```diff
@@ -869,15 +869,15 @@ class ConstraintLocatorBuilder {
   }
 
   /// Checks whether this locator is describing an anchor that produces a value of a
-  /// callable type that could be implicit called using "()" either by defining
+  /// callable type that could be explicit called using "()" either by defining
```

Comment:
?

Reply:
Sorry, I was just confused about what word would make sense to describe that "we are checking if the user could insert a explicit () call on source to produce the expected type". For that, the value of the type has to be implicit callable to produce the value. I just reverted :)

Reply:
Is this terminology, “explicit call” and “implicitly callable,” used anywhere? It seems confusing to me: I don’t understand what’s explicit or implicit about this. My understanding is that this is all about calling things that are callable.

Reply:
> My understanding is that this is all about calling things that are callable.

Yes, is just that. I think in the sense of the comment we can just use `could be called` because that is what we trying to verify.